src/lib/agent/PolykeyAgent.ts

```diff
@@ -303,7 +372,7 @@ class PolykeyAgent {
       throw Error(`node path already exists: '${nodePath}'`);
     }
 
-    const { name, email, passphrase, nbits } = NewNodeRequestMessage.decode(request);
+    const { name, email, passphrase, nbits } = NewNodeRequestMessage.decodeDelimited(request);
```

Comment:
If this is a new node, I believe we only need an arbitrary identifier, it doesn't need to be name or email. Any identifier is fine.

Also we should do the same as wireguard and fix all crypto parameters and instead of protocol negotiation, set it by version.

This means the GRPC protocol needs to always answer by version. Ask @nzhang about this.

Reply:
ahh yes this has been a long time coming, I will take some time now to make it so.

actually, kbpgp only has a `userId` field. But in their example they split it up with `Full Name <email> (comment)`. We can just ask the user to provide something like a `polykeyId`

Reply:
done.